tests/models_tests/models_test.go

```diff
@@ -814,18 +814,27 @@ func TestRetrieveDataBySourceIdAndCohortIdAndConceptIdsOrderedByPersonId(t *test
 			t.Errorf("Expected some cohort data")
 		}
 		var previousPersonId int64 = -1
+		defaultHareValue := map[string]bool{"": true, "non-Hispanic Asian": true, "non-Hispanic Black": true, "non-Hispanic White": true, "Hispanic": true}
```

Comment:
why is `"": true` ?

Reply:
There is a hare concept observation row with empty value_as_concept_id in the test data set.

Reply:
https://github.com/uc-cdis/cohort-middleware/blob/944e952a97fd3085801a32c9781da0fab81b59a9/tests/setup_local_db/test_data_results_and_cdm.sql#L119

Reply:
I modified the test to only allow empty value once.